collectors_app.php

```diff
@@ -21,68 +21,108 @@
 
 // html form ✅
 // $_GET form data ✅
-// move data to DB
+// move data to DB ✅
 
-//----------------------------------------------------------------------------------------------------------------------empty socks array
-$socks = [];
+// trainer tasks:
 
-//----------------------------------------------------------------------------------------------------------------------connect to DB
-$db = new PDO(
-    'mysql:host=DB;dbname=collection',
-    'root',
-    'password'
-);
-//----------------------------------------------------------------------------------------------------------------------defaulting
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-//----------------------------------------------------------------------------------------------------------------------preparing query
-$query = $db->prepare('SELECT * FROM `socks`');
-//----------------------------------------------------------------------------------------------------------------------execute query
-$result = $query->execute();
+// validation of inputs:
+// DB table for sizes, color, pattern ✅
+// dropdowns that populate colors, sizes, patterns ✅
 
-if ($result) {
-    $socks = $query->fetchAll();//--------------------------------------------------------------------------------------REASSIGN PLAYERS ARRAY
-} else {
-    echo 'not working';
+//----------------------------------------------------------------------------------------------------------------------dropdowns
+//----------------------------------------------------------------------------------------------------------------------connect DB function
+function connectDB() {
+    $db = new PDO(
+        'mysql:host=DB;dbname=collection',
+        'root',
+        'password'
+    );
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
 }
```

Comment:
Lets get this into it's own file now, time to tidy up a bit!

Reply:
Does this mean to use 'require' to link a separate file?